src/filesystem/impls/appshell/node/node_modules/fsevents_win/fsevents_win.cpp

```diff
@@ -337,7 +339,7 @@ DWORD WINAPI NodeFSEvents::Run(LPVOID lpData)
 	{
 		// iterate watching for directory changes until we're signalled to stop
 		BOOL bContinue = TRUE;
```

Comment:
Hi @bchintx there are several points with removing `bContinue` and leaving the rest untouched.
1. on line 247 there are `FILE_FLAG_OVERLAPPED` is set meaning  `ReadDirectoryChangesSW` is working in async mode. 
2. MSDN says that for asynchronous mode returning value is a result of task submission, false means we run out of some resources, handles, stack, ram, etc. 
3. Submitting new requests, e.g. calling `ReadDirectoryChangesSW` in that situation (if that ever happen i believe in low resource condition there will be lots of problems) makes situation even worse: I'm not sure whether that function will issue `SwitchToThread` call, if not we will have one cpu core load spot until low resource condition go away.
4. Further reading of possible error codes reported by `ReadDirectoryChangesW()` makes me think they all are not recoverable, so no point to keep that loop

I just wanted to say that if we simply ignore that error we have to think about possible short loop that burns CPU, and at least add `SwitchToThread()`, `Sleep(0)`, etc, if we decided to not abort watcher thread.